crates/swc_ecma_parser/src/parser/jsx.rs

```diff
@@ -143,6 +143,9 @@ impl<I: Tokens> Parser<I> {
         let expr = if is!(self, '}') {
             self.parse_jsx_empty_expr().map(JSXExpr::JSXEmptyExpr)?
         } else {
+            if is!(self, "...") {
```

Comment:
Typescript parse this `dotdotdotToken` as an optional token but they parse still as a `JSXExpression`, I found that this token seems not to have effects on code generation, so I just skip it in the parsing phase.